packages/@o3r/rules-engine/src/fixtures/jasmine/rules-engine.runner.service.fixture.jasmine.ts

```diff
@@ -1,19 +1,14 @@
-import type { RulesEngineService } from '@o3r/rules-engine';
+import type { RulesEngineRunnerService } from '@o3r/rules-engine';
 
-export class RulesEngineServiceFixture implements Readonly<Partial<RulesEngineService>> {
+/** @deprecated use RulesEngineRunnerServiceFixture instead, will be removed in Otter v10 */
```

Comment:
```suggestion
/** @deprecated use RulesEngineRunnerServiceFixture instead, will be removed in Otter v12 */
```
There are several like this :)